Extensions/bezier/bezier.lisp

```diff
@@ -109,10 +109,15 @@
       record
     (let ((dx (- nx x1))
           (dy (- ny y1)))
-      (let ((tr (make-instance 'climi::standard-translation :dx dx :dy dy)))
+      (let ((current-translation (or
+                                  (output-record-translation record)
+                                  (make-instance 'climi::standard-translation :dx 0 :dy 0)))
+            (tr (make-instance 'climi::standard-translation :dx dx :dy dy)))
         (multiple-value-prog1
             (call-next-method))
```

Comment:
what's the point of mvp1 here? it seems that it is not returned.

Reply:
I don't know, I didn't touch that. I guess it returned the new coordinates before the change wrt. translations was introduced?